### PR TITLE
git-ls-files: add page

### PR DESCRIPTION
**Pull request by sanehab**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For #3953

### CODE REVIEW

File: pages/common/git-ls-files.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# git ls-files
+
+> Git command for showing information about files in the index and the working tree.
+> More information: <https://git-scm.com/docs/git-ls-files>.
+
+- Show deleted files in the output:
+
+`git ls-files --deleted`
+
+- Show modified files in the output - deleted files are also part of the modified files:
```
Comment (navarroaxel) [inline pages/common/git-ls-files.md:10]: Maybe this is better:
```suggestion
- Show modified and deleted files:
```

File: pages/common/git-ls-files.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# git ls-files
+
+> Git command for showing information about files in the index and the working tree.
+> More information: <https://git-scm.com/docs/git-ls-files>.
+
+- Show deleted files in the output:
```
Comment (navarroaxel) [inline pages/common/git-ls-files.md:6]:
```suggestion
- Show deleted files:
```

File: pages/common/git-ls-files.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+- Show modified files in the output - deleted files are also part of the modified files:
+
+`git ls-files --modified`
+
+- Show ignored and untracked files:
+
+`git ls-files --others`
+
+
+
```
Comment (navarroaxel) [inline pages/common/git-ls-files.md:19]: You should delete this empty lines to pass the linter check:
```suggestion
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @sanehab, I left some comments.

**Review by navarroaxel — APPROVED** [review]

File: pages/common/git-ls-files.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# git ls-files
+
+> Git command for showing information about files in the index and the working tree.
```
Comment (sbrl) [inline pages/common/git-ls-files.md:3]: Perhaps this will read better?
```suggestion
> Show information about files in the index and the working tree.
```

**Review by sbrl — COMMENTED** [review]
Nice one! I've left a comment below for you to review.

File: pages/common/git-ls-files.md (hunk 4 above)
Comment (sanehab) [reply pages/common/git-ls-files.md]: @sbrl I do agree that this is better, however notice that some git commands have descriptions like this, and this is where i get this template from :)

**Review by sbrl — APPROVED** [review]

**Pull request merged**